.gitignore

```diff
@@ -4,3 +4,4 @@
 config.json
 *.pyc
 *.iml
+ansible/my-inventory
```

Comment:
I have a `ansible/inventory.bak` which achieves the same. `*bak` is already in the `.gitignore` file.